### PR TITLE
Security indicator example

### PR DESCRIPTION
**Pull request by KPostOffice**
## Related Issues and Dependencies

https://github.com/thoth-station/adviser/issues/1178

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -80,3 +80,24 @@ To browse Thoth's logs during or after the adviser run:
...
 
   thamos log
+
+
+Security
+========
+
+One of the options that can be set when getting advice is setting the :code:`recommendation_type`.  On Thoth's adviser
+adviser this indicates different steps that should be included when scoring packages.  By setting 
+:code:`recommendation_type == security` adviser will include steps such as checking for CVE and scoring based on Thoth's
```
Comment (fridex) [inline README.rst]: To be more accurate s/steps/pipeline units/g - here and above. Pipeline units of any time can be added based on the recommendation type configuration option.

File: .thoth.yaml [hunk 2]
```diff
@@ -8,7 +8,7 @@ runtime_environments:
       name: ubi
       version: "8"
     python_version: "3.6"
-    recommendation_type: stable
+    recommendation_type: security
```
Comment (fridex) [inline .thoth.yaml:11]: To demo the difference, would it make sense to put this under a directory (e.g. `security`) and have another directory aside (e.g. `latest`) so we show the difference in output?

**Review by fridex — COMMENTED** [review]
lgtm, just two minor comments :+1:

**Comment by goern** [comment]
@KPostOffice @fridex is this good to merge?

File: .thoth.yaml (hunk 2 above)
Comment (KPostOffice) [reply .thoth.yaml:11]: I'm pulling this to a different branch. I could create another branch that does a `latest` advise

**Review by fridex — APPROVED** [review]
lgtm!! :+1:

**Pull request merged**